src/styles/utils/reset.scss

```diff
@@ -0,0 +1,6 @@
+*,
+p {

```

Comment:
If you set styles for everything through `*`, you don't need to select any tags separately.